content/source/docs/enterprise/users-teams-organizations/users.html.md

```diff
@@ -4,47 +4,85 @@ page_title: "Users - Terraform Enterprise"
+
+After you create a new user account, TFE immediately takes you to a page where you can create a new organization.
+
+![screenshot: the new organization page](./images/org-new.png)
+
+- If you are the first TFE user in your organization, use this page to create your TFE organization. See [Organizations][] for more information.
+- If you intend to join an existing organization, do not create a new one; instead, send your username to one of your TFE organization's owners and ask them to add you to a team. Once they have brought you into the organization, you can reload the page to begin using TFE.
 
 ## Team and Organization Membership
 
-Once a user has an account, an organization owner can add them to the
-organization by adding them to any of the organization's teams.
+To add a user to an organization, a member of that organization's owners team must add them to one or more teams. See [Teams][] for more information.
 
-To add a user to a team, navigate to the "Teams" section in your organization
-settings. Create a team if it does not already exist, or add the user to an
-existing team.
+Adding a user to a team requires only their username.
 
 ## User Settings
 
+-> **API:** See the [Account API](../api/account.html).
+
+TFE users can manage many of their own account details, including email address, password, API tokens, and two-factor authentication.
+
 To reach your user settings page, click the user icon in the upper right corner and choose "User Settings" from the menu.
 
-Once on this page, you can change your password and generate or revoke API tokens.
+Once on this page, can use the navigation on the left to choose which settings to manage.
 
 ![The user settings menu item, in the upper right menu.](./images/user-settings.png)
 
+### Profile
+
+TFE user profiles are very small, consisting only of a username and an email address. You can change either of these from the "Profile" page of the user settings.
+
+~> **Important:** Changing your username can cause important operations to fail. This is because it is used in URL paths to various resources. If external systems make requests to these resources, you'll need to update them prior to making a change.
+
+Additionally, TFE uses [Gravatar](http://en.gravatar.com) to display a user icon if you have associated one with your email address. For details about changing your user icon, see [Gravatar's documentation](http://en.gravatar.com/support/).
+
 ### Password Management
 
-Users manage their own passwords. To change your password, click the "Security" tab of the user settings page. You'll need to confirm your current password, and enter your new password twice.
+Users manage their own passwords. To change your password, click the "Password" page of the user settings. You'll need to confirm your current password, and enter your new password twice.
 
 Password management isn't available if your organization uses [SAML single sign on](../saml/index.html).
```

Comment:
We probably shouldn't use "organization" here (although I see we were before) because it's overloading a term. I'd say "If your TFE installation uses".

Reply:
Oh heck, you're right, it slipped my mind that you can't configure that per-org.